src/main/java/bitbucketpullrequestbuilder/bitbucketpullrequestbuilder/BitbucketBuildTrigger.java

```diff
@@ -54,6 +54,7 @@
     private final boolean approveIfSuccess;
     private final boolean cancelOutdatedJobs;
     private final String commentTrigger;
+    private final boolean declineIfFailure;
```

Comment:
I doubt approving makes sense. There is build status, you can use it. Approval is for humans. Same for declining. The build may fail for an unrelated reason, declining PR for that would be harsh :)

Reply:
While I agree with your position (declining / approval is mostly for humans), *IF* we allow a decline, which is kinda a nuclear option, IMHO, then why not allow approvals?

And, since more than one person has submitted patches to do approval / declines, then people are obviously wanting this in their process - why not let them have it. You don't have to click the button ;)

Personally, I see more need for approvals than for declines, but, YMMV